.travis.yml

```diff
@@ -51,14 +51,15 @@ install:
           # want cmake newer than by default in travis osx
           brew upgrade cmake
           export CMAKE_FRAMEWORK_PATH="$CMAKE_FRAMEWORK_PATH:$(brew --prefix qt5)"
+          ADDITIONAL_LINK_ARGS="-stdlib=libc++ -mmacosx-version-min=10.7"
```

Comment:
I think this should be fixed not here, but in CMakeLists.txt.
Everyone should be able to build on OS X, not only Travis.